magnum-zio/src/main/scala/com/augustnagro/magnum/zio_integration/zio.scala

```diff
@@ -0,0 +1,128 @@
+package com.augustnagro.magnum.zio_integration
```

Comment:
Please no underscore in package names. Either `.integration.zio` or just `.zio` I'd say. I prefer the latter

Reply:
Why not? The issue with naming it `zio` is that it creates really annoying conflicts in the imports.

This code:
```scala
import com.augustnagro.magnum.zio
import zio.Task
```
will fail to compile because `Task` is not in the `com.augustnagro.magnum.zio` package.
It forces people to do this manually:
```scala
import com.augustnagro.magnum.zio
import _root_.zio.Task
```
Really not great UX.

I initially called it `com.augustnagro.magnum.zio` but even just coding this PR it was annoying

Reply:
`.zio.integration`
Underscore is against JVM package name convention

Reply:
What about `magzio`

Reply:
Done 🙂

Reply:
Some tests are failing. I'll have a look at that after work

Reply:
You did not copy the resource files into the zio module
```scala
    val carSql =
      Files.readString(Path.of(getClass.getResource("/pg-car.sql").toURI))
    val personSql =
      Files.readString(Path.of(getClass.getResource("/pg-person.sql").toURI))
    val bigDecSql =
      Files.readString(Path.of(getClass.getResource("/pg-bigdec.sql").toURI))
    val noIdSql =
      Files.readString(Path.of(getClass.getResource("/pg-no-id.sql").toURI))
  ```

Reply:
> You did not copy the resource files into the zio module

I don't need as the `zio` module depends on the `magnum` module with `"test->test"`

I had to re-write the tests as they have changed :)